src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let price = 0;
+
+  if (days >= 7) {
+    price = days * 40 - 50;
```

Comment:
you should add variables for 50 & 20
`  const smallDiscount = 20;`
`  const bigDiscount = 50`